Sources/Cryptor/Digest.swift

```diff
@@ -51,17 +54,28 @@ public class Digest: Updatable {
     public enum Algorithm {
 		
         /// Message Digest 2 See: http://en.wikipedia.org/wiki/MD2_(cryptography)
+        @available(iOS, introduced: 2.0, obsoleted: 13.0, message: "This function is cryptographically broken and should not be used in security contexts. Clients should migrate to SHA256 (or stronger).")
```

Comment:
nitpick: Personally I'm not fond of the word "broken" because it implies that it doesn't work.  But I understand it is a common industry phrasing.   I like Apple's description of just plainly saying it is [not cryptographically secure](https://developer.apple.com/documentation/cryptokit/insecure/md5).   (Some more detailed discussion on the topic if you're interested: https://www.quora.com/What-is-broken-cryptography)

But I'm ok with leaving your comment here as is.

Reply:
Updated to say "insecure" instead of "broken"